node/service/src/chain_spec/frequency_local.rs

```diff
@@ -21,73 +21,15 @@ pub fn template_session_keys(keys: AuraId) -> frequency_rococo_runtime::SessionK
 	frequency_rococo_runtime::SessionKeys { aura: keys }
 }
 
-pub fn development_config() -> ChainSpec {
```

Comment:
Where does a development genesis config live now?

Reply:
It was same as testnet, got rid of one as we either run frequency as instan seal or as rococo collator locally